net.veloren.airshipper.yaml

```diff
@@ -13,30 +10,60 @@ finish-args:
   - --share=network
   - --share=ipc
   - --device=dri
-build-options : {
-  append-path : "/usr/lib/sdk/rust-nightly/bin"
-}
+build-options:
+  append-path: "/app/sdk/rust-nightly/bin"
+cleanup:
+  - /sdk
 
 modules:
+  - name: rust
+    buildsystem: simple
+    sources:
+      - type: archive
+        only-arches:
+          - aarch64
+        dest: rust
+        url: https://static.rust-lang.org/dist/2024-01-15/rust-nightly-aarch64-unknown-linux-gnu.tar.xz
+        sha256: 946ca47cb7797a94abf4b83b3410d62f17e47dca2bf1841b75ce36bcde72bb10
+        x-checker-data:
+          type: rust
+          package: rust
+          channel: nightly
+          target: aarch64-unknown-linux-gnu
```

Comment:
```suggestion
```

You don't need the x-checker part, since you want to keep the nightly version fixed. This is going to send a PR daily.